src/ucp/core/ucp_worker.c

```diff
@@ -43,6 +43,12 @@
 
 #define UCP_WORKER_MAX_DEBUG_STRING_SIZE 200
 
+
+#define UCP_WIFACE_FMT "iface %p (" UCT_TL_RESOURCE_DESC_FMT ")"
+#define UCP_WIFACE_ARG(_wiface, _resource) \
+    (_wiface)->iface, UCT_TL_RESOURCE_DESC_ARG(_resource)
```

Comment:
the macro should not accept "resource". it could find it from the wife object

Reply:
Addressed by https://github.com/openucx/ucx/pull/8986/commits/18c52b9863cb5701036a7ed1444e51bc671c0baf